classes/Http/Form/ForgotFormType.php

```diff
@@ -19,7 +19,7 @@
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\Validator\Constraints as Assert;
 
-class ForgotForm extends AbstractType
+final class ForgotFormType extends AbstractType
```

Comment:
Renamed this class as it's not actually a form, but a definition of a form, right?

Reply:
Since this change is unrelated to the rest of the PR, maybe move it to another PR?